test/jsx.tsx

```diff
@@ -38,14 +38,16 @@ describe('snabbdom', function() {
       const showLogin = false;
       const showCaptcha = false;
       const loginAttempts = 0;
-      const userName = undefined;
+      const userName = ``;
```

Comment:
I see below that an empty string is ignored instead of becoming an empty DOM text node. Is this the right thing to do?

Reply:
In JSX an empty string is ignored. https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

Reply:
@RyanHirsch thank you for the input.

Seems right.

Reply:
let me change it back to `if (child !== undefined && child !== null && child !== false) {`

!child may mean too many things in JS, NaNs are also falsey and we may want that.